### PR TITLE
KEYCLOAK-16486 Update app-angular2 QS

### PR DESCRIPTION
**Pull request by mhajas**
- newer Keycloak JS adapter
- using chrome headless in testsuite

### CODE REVIEW

File: app-angular2/src/main/webapp/package.json
Comment (vmuzikar) [inline app-angular2/src/main/webapp/package.json:26]: I wonder, couldn't we use `keycloak.js` from the server instead of hardcoding a preset version here?

**Review by vmuzikar — CHANGES_REQUESTED** [review]
@mhajas Sorry for late review. Changes look good, added one comment, but I'm no JS/Angular expert. @edewit Could you please give this a quick look too?

Could you please also update the README with instructions on how to run the tests, similarly to e.g. [here](https://github.com/keycloak/keycloak-quickstarts/blob/latest/user-storage-simple/README.md#integration-test-of-the-quickstart)?

**Comment by vmuzikar** [comment]
@mhajas One last question. You might have already mentioned this somewhere but did you try to run it with GH Actions (#225)?

**Comment by mhajas** [comment]
@vmuzikar, I would say I did, but I am not sure. We can wait for it to be merged and then rebase this to be sure, WDYT?

**Comment by vmuzikar** [comment]
> @vmuzikar, I would say I did, but I am not sure. We can wait for it to be merged and then rebase this to be sure, WDYT?

@mhajas Ok, works for me.

File: app-angular2/src/main/webapp/package.json
Comment (mhajas) [reply app-angular2/src/main/webapp/package.json:26]: I somehow missed your comment, sorry. Well I am not sure it is a good idea. I am not really an expert in angular, but I believe this is the way how the community will use it. Maybe we should change it to `latest` so that we always test with latest realease of keycloak-js. WDYT?

Comment (vmuzikar) [reply app-angular2/src/main/webapp/package.json:26]: Ok, I see your point. Even though it might not be ideal (let's say we're testing a PR to the KC repo which makes a breaking change to the adapter, this solution wouldn't catch it), let's use `latest`.

File: .github/scripts/export-keycloak-version.sh
Comment (vmuzikar) [inline .github/scripts/export-keycloak-version.sh:7]: I think we could use a similar command as below:
```suggestion
  export KEYCLOAK_VERSION=$(mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
```

**Review by vmuzikar — COMMENTED** [review]
@mhajas Thanks for the update! Added one last nitpick. :)

I think it might even work with https://github.com/keycloak/keycloak/pull/7929 (I updated it to checkout KC repo to a dir with the same name as here).

File: .github/scripts/export-keycloak-version.sh
Comment (mhajas) [reply .github/scripts/export-keycloak-version.sh]: Done

**Review by vmuzikar — APPROVED** [review]
@mhajas Thanks! Now let's wait for a review from @ssilvert or @edewit. :)

**Comment by mhajas** [comment]
Thanks @vmuzikar!

**Comment by edewit** [comment]
I'll leave this for @ssilvert to try out I'm a bit out of my depth on it

**Review by abstractj — APPROVED** [review]

**Pull request merged**
